packages/atlas/src/config/videoFilter.ts

```diff
@@ -0,0 +1,13 @@
+import { VideoWhereInput } from '@/api/queries'
+
+export const videoFilter: VideoWhereInput = {
+  isCensored_eq: false,
+  isPublic_eq: true,
+  thumbnailPhoto: {
+    isAccepted_eq: true,
+  },
+  media: {
+    isAccepted_eq: true,
+  },
+  NOT: [{ id_in: [] }, { thumbnailPhoto: { id_in: [] } }, { media: { id_in: [] } }],
+}
```

Comment:
I still can go to the video which is blocked. Didn't we agree that we should hide this from a user as well?